Project/core/src/main/java/Controller/GameController.java

```diff
@@ -0,0 +1,118 @@
+package Controller;
+
+import Model.Player;
+import Model.Recruiter;
+import Model.RougeAgent;
+import Model.Game;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameController {
+    // Check for new turn
+    // Think this would be better as view calling it to move from one turn to the
+    // next as needed
+    // Update recruits if odd turn
+    // Activate player
+    // Check win
+
+    private boolean recruiterTurn = true;
+    private Game gameState;
```

Comment:
Should the GameController contain and store the gameState? Or should it be somewhere outside in main?